orttraining/orttraining/core/session/training_session.h

```diff
@@ -182,6 +182,8 @@ class TrainingSession : public InferenceSession {
       // cut_list contains the list of CutInfo to make the graph partitions.
       // cut_list[i] contains the CutInfo to make the partition between stage i and stage i+1
       std::vector<CutInfo> cut_list;
+      // Alternative for partition. We map operators to rank ids. 
+      std::map<std::string, int> op_id_to_rank;
```

Comment:
Nit: I would suggest to rename this to node_id_to_rank. Components of the graph are referred to as nodes in this file.